packages/app-desktop/gui/editorCommandNames.ts

```diff
@@ -0,0 +1,21 @@
+export default function() {
+	return [
+		'editor.deleteLine',
+		'editor.undo',
+		'editor.redo',
+		'editor.goDocStart',
+		'editor.goDocEnd',
+		'editor.goGroupLeft',
+		'editor.goGroupRight',
+		'editor.goLineStart',
+		'editor.goLineEnd',
+		'editor.delGroupBefore',
+		'editor.delGroupAfter',
+		'editor.indentLess',
+		'editor.indentMore',
+		'editor.toggleComment',
+		'editor.sortSelectedLines',
+		'editor.swapLineUp',
+		'editor.swapLineDown',
+	];
+}
```

Comment:
Could this array be dynamically built based on the command prefix? That way we won't need to remember to update this list when we add a new command.

Reply:
Also should the other editor commands, without a prefix, be included here? If not, please could you add a comment explaining why (or maybe by renaming the file so that it's clear why some commands are here and not others).

Reply:
Good point, these are editor commands that make sense to have key bindings for, but aren't present in the menu. Thinking about this, it might actually make the most sense to add these commands to the menu.

I'm going to add them now and perhaps we can discuss more then.